frontend/src/pages/Inspection/MetaData.jsx

```diff
@@ -1,92 +1,228 @@
   } = data;
+  const dispatch = useDispatch();
+
+  const [likes, setLikes] = useState(0);
+  const [isLiked, setLiked] = useState(false);
+
+  useState(() => {
+    const internal = async () => {
+      try {
+        const res = await fetch(`/api/Metadata/${uuid}/likes`);
+        const { likeCount, liked } = await res.json();
+        setLikes(likeCount);
+        setLiked(liked);
+      } catch (err) {
+        const { status } = err;
+        if (status === 404) {
+          dispatch(alertActions.error('Could not fetch likes for dataset.'));
+        } else {
+          dispatch(alertActions.error('Failed to fetch likes. Please try again later.'));
+        }
+      }
+    };
+    internal();
+  }, []);
+
+  const handleLike = async () => {
+    try {
+      // eslint-disable-next-line no-irregular-whitespace
+      const userSelector = useSelector((state) => state.user);
+      const { token } = userSelector.user;
+      await fetch(`/api/SOMETHING-FOR-ADDING-LIKE/${uuid}/${token}`);
+    } catch (err) {
+      dispatch(alertActions.error('Something went wrong when adding/removing favourite.'));
+    }
+  };
 
   return (
     <Wrapper>
-      <h2>
-        Showing metadata about dataset
-        {` ${metadataTypeName} from ${municipalityName}`}
-      </h2>
-      <DateLine>
-        Published
-        {` ${date}`}
-      </DateLine>
-      <Description>
-        {description}
-      </Description>
-      <div>
-        {tags.map((tag) => <Tag key={tag}>{tag}</Tag>)}
-      </div>
-      <div>
-        <Source href={url}>
-          {`[${url}]`}
-          <FileFormat>
-            <p>
-              {formatName}
-            </p>
-          </FileFormat>
-        </Source>
-      </div>
-      <hr />
-      <div>
-        {
-          experiencePostGuid
-            ? <Link to={`/articles/${experiencePostGuid}`}>Read Experience Article</Link>
+      <MetadataContent>
+        <ReleaseStateLabel releaseState={releaseState} />
+        <h2>
+          Showing metadata about dataset
+          {` ${metadataTypeName} from ${municipalityName}`}
+        </h2>
+        <DateLine>
+          Published
+          {` ${date}`}
+        </DateLine>
+        <Description>
+          {description}
+        </Description>
+        <div>
+          {tags.map((tag) => <Tag key={tag}>{tag}</Tag>)}
+        </div>
+        <div>
+          <Source href={url}>
+            {`[${url}]`}
+            <FileFormat>
+              <p>
+                {formatName}
+              </p>
+            </FileFormat>
+          </Source>
+        </div>
+      </MetadataContent>
+      <MetadataToolbar>
+        <FavouriteButton onClick={handleLike}>
+          {isLiked ? <StarFullStyled /> : <StarEmptyStyled />}
+          <p>Interesting</p>
+          <LikeCounter>{likes}</LikeCounter>
+        </FavouriteButton>
+        <div>
+          {
+          experiencePostGuid !== undefined
```

Comment:
There's no reason to have " experiencePostGuid !== undefined".  "!experiencePostGuid" works fine as null or undefined will be equal to false. Not to mention that this doesn't work when it's null.

<img width="830" alt="Skjermbilde 2020-02-29 kl  12 32 26" src="https://user-images.githubusercontent.com/15128509/75606719-e2625480-5aef-11ea-9537-b02e60e13947.png">